src/AdminConsole/Services/PasswordlessManagement/IPasswordlessManagementClient.cs

```diff
@@ -22,4 +22,5 @@ public interface IPasswordlessManagementClient
     Task<GetAppIdAvailabilityResponse> IsApplicationIdAvailableAsync(GetAppIdAvailabilityRequest request);
     Task EnableMagicLinksAsync(string appId, string performedBy);
     Task DisableMagicLinksAsync(string appId, string performedBy);
```

Comment:
Would these be removed?

Reply:
probably. unless we think its useful to keep them around

Reply:
I vote remove